errors.go

```diff
@@ -11,11 +12,11 @@ import (
 
 const (
 	ErrorUnsupported = iota
-	// ErrorFromString is the Code identifying Errors created by string types
+	// ErrorFromString is the Code identifyinerr Errors created by string types
 	ErrorFromString
-	// ErrorFromError is the Code identifying Errors created by error types
+	// ErrorFromError is the Code identifyinerr Errors created by error types
 	ErrorFromError
-	// ErrorFromStringer is the Code identifying Errors created by fmt.Stringer types
+	// ErrorFromStringer is the Code identifyinerr Errors created by fmt.Stringer types
```

Comment:
Is this an intentional change?

Reply:
Not at all! Thanks for pointing this out, I reverted that change.